certificate-service/app/src/main/resources/fk7472/schematron/itfp.v1.sch

```diff
@@ -14,25 +14,25 @@
 
   <iso:pattern id="intyg">
     <iso:rule context="//rg:intyg">
-      <iso:assert test="count(gn:svar[@id='2']) = 1">
+      <iso:assert test="count(gn:svar[@id='55']) = 1">
         Måsta ange ett 'Barnets diagnos eller symtom' svar
       </iso:assert>
-      <iso:assert test="count(gn:svar[@id='3']) = 1">
+      <iso:assert test="count(gn:svar[@id='56']) = 1">
         Måste ange ett 'Period som barnet inte bör vårdas i ordinarie tillsynsform' svar
       </iso:assert>
     </iso:rule>
   </iso:pattern>
 
   <iso:pattern id="q2">
```

Comment:
It might be a good idea to update the patterns so they have an matching id aswell!